docs/maintainers/maintainers-guide.md

```diff
@@ -0,0 +1,152 @@
+
+There are three tabs that provide you with more information on these repositories.
+
+### Overview
+The Overview view gives you a high-level view of the contibutors in your list, including the total number of commits in the last 30 days and the types of contributors on your list: active, new, and alumni.
+
+### Activity
+The Activity tab gives you a graph view with more detailed information on each of the contributors, including type of activity, repositories they've contributed to, and how they compare to each other. 
+
+You can filter your list by all contributors, active contributors, new contributors, and alumni contributors.
+
+![list graph](../../static/img/list-graph.png)
+
+### Contributors
+The contributors view, gives you the list of the contributors and includes information like activity level, last repository contributed to, the date of their last contribution, time zone, programming languages used, and number of contributions.
+
+ To learn more about these features, see [Understanding Lists Data](#understanding-lists-data).
+
+## Highlights: Connecting and Recognizing Contributions
+Leveraging OpenSauced's Highlights feature is a great way for maintainers to recognize and showcase the valuable contributions made to your open source projects, to curate and share key achievements, contributions, and milestones, and bring visibility to the hard work of your community. 
+
+Highlights can be used to:
+- publicize issues on your repository that you'd like to match with contributors
+- share Dev.to blog posts you've written
+- celebrate PRs to your project
+
+### Creating a New Highlight
+There are two ways to create a new highlight:
+1. On the [main feed](https://app.opensauced.pizza/feed), click the input that says "Post a highlight to show your work!"
+
+2. From your profile, click the input that says "Post a highlight to show your work!"
+
+After you've clicked the input, you'll see a form that allows you to create a new highlight.
+
+There are two ways to create a highlight:
+1. Paste the URL to your PR, Issue, or Dev.to blog post (shown in blue below).
+2. Choose from a list of suggestions (shown in orange below).
+
+![highlight suggestions](../../static/img/highlights.png)
+
+If you're adding a blog post, you'll need to associate it to a repository by clicking the "Add a repo" button.
+
+Finally, it's time to add the details of your highlight. You can add your own description or use the auto-summarize function to generate a description of.
+
+![gif of creating a highlight](../../static/gif/highlight.gif)
+
+### Writing an Effective Highlight
+Writing an effective highlight is key to attracting contributors to your project. Here are some tips to help you write an effective highlight:
+
+#### Issues
+1. **Focus on the Problem Solved**: Begin by succinctly describing the problem or challenge the issue addresses. This sets the context for why it was important.
```

Comment:
Should we do lower case for the bold ones? Or do we want to treat them like a title?

```suggestion
1. **Focus on the problem solved**: Begin by succinctly describing the problem or challenge the issue addresses. This sets the context for why it was important.
```

Reply:
I was treating it like a title, but I'm fine with whatever as long as we stay consistent.

Reply:
If it is a common `ol` — for example, like instruction steps, etc. — I'd say let's do lower case.

If we want to make it a title, then we apply heading like this:

```markdown
##### 1. Focus on the Problem Solved

Begin by succinctly describing the problem or challenge the issue addresses. This sets the context for why it was important.
```

Reply:
Oh, and I changed the word "succinctly" to "briefly" because I think it's easier to understand for non-native English speaker (I had to look up the meaning of it 😅)

Reply:
Thanks! That's really helpful!